src/test/java/org/opentripplanner/transit/raptor/speed_test/ResultPrinter.java

```diff
@@ -204,8 +205,25 @@ private static void printProfileResultLine(String label, List<Integer> v, int la
         "[ " +
         v.stream().map(it -> String.format("%4d", it)).reduce((a, b) -> a + ", " + b).orElse("") +
         " ]";
-      double avg = v.stream().mapToInt(it -> it).average().orElse(0d);
-      System.err.printf(" ==> %-" + labelMaxLen + "s : %s Avg: %4.1f%n", label, values, avg);
+      IntSummaryStatistics statistics = v.stream().mapToInt(it -> it).summaryStatistics();
+      double average = statistics.getAverage();
+      double standardDeviation = 0.0;
+
+      for (double num : v) {
+        standardDeviation += Math.pow(num - average, 2);
+      }
+
+      standardDeviation = Math.sqrt(standardDeviation / v.size());
```

Comment:
This can be moved to IntUtils (which should be moved to `org.opentripplanner.util.lang`) or at least extracted into a static utility function.